strawberry/sentinel.py

```diff
@@ -0,0 +1,95 @@
+"""Reference implementation of [PEP-0661](https://www.python.org/dev/peps/pep-0661/)
+
+See https://github.com/taleinat/python-stdlib-sentinels
+"""
```

Comment:
isn't there a package for this?

if so we should use, otherwise we need to include the license :)

Reply:
There's a https://pypi.org/project/sentinel/ and a https://pypi.org/project/sentinels/ library, neither of them are connected to the PEP as far as I can tell, but they both claim to have very similar features. I don't see a license of the reference implementation github, but the PEP itself has "This document is placed in the public domain or under the CC0-1.0-Universal license, whichever is more permissive."

I can switch this to one of the libraries mentioned if you want, I agree it doesn't make much sense to maintain our own implementation.

Reply:
Licensing for the `python-stdlib-sentinels` might be a bit tricky because it doesn't include an explicit LICENSE in the repo

Reply:
Yeah I'll go ahead and swap to one of the libraries